hyperspy/__init__.py

```diff
@@ -39,7 +26,6 @@
 
 import Release
 
+__all__ = ["api"]
```

Comment:
Sorry for misunderstanding your question. IPython's autocompletion works fine, this should only impact what is imported by `from hyperspy import *`.